backend/src/apiserver/template/template_test.go

```diff
@@ -300,3 +246,51 @@ func TestModelToCRDTrigger_Cron(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, expectedCRDTrigger, actualCRDTrigger)
 }
+
+func loadYaml(t *testing.T, path string) string {
+	res, err := ioutil.ReadFile(path)
+	if err != nil {
+		t.Error(err)
+	}
+	return string(res)
+}
+
+func TestIsKubernetesExecutorConfig(t *testing.T) {
+	template := loadYaml(t, "testdata/pipeline_with_volume.yaml")
+	splitTemplate := strings.Split(template, "\n---\n")
+	assert.True(t, isKubernetesExecutorConfig([]byte(splitTemplate[1])))
```

Comment:
Can we also add `assert.False` to make sure the function works both ways.

Reply:
Good idea. I will add this.

Reply:
done.